docs/conf/modules.conf.example

```diff
@@ -1589,25 +1607,25 @@
 
 #-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#
 # Servprotect module: Provides support for Austhex style +k /
-# UnrealIRCD +S services mode.
+# UnrealIRCd +S services mode.
```

Comment:
We should explain what the mode does here, pointing to another implementation is not very helpful